### PR TITLE
uoffset harmonisation

### PR DESCRIPTION
**Pull request by abuts**
Its relatively small changes to ensure all uoffset values in the Horace are expressed in hkl.

### CODE REVIEW

File: _test/test_sqw/test_mask.m [hunk 1]
```diff
@@ -259,13 +259,13 @@ function test_num_pix_has_been_reduced_by_correct_amount_paged_pix_3d(obj)
         function test_img_db_range_recalculated_after_mask_3d(obj)
             original_img_db_range = obj.sqw_3d.data.img_db_range;
             img_db_range_diff = abs(original_img_db_range - obj.masked_3d.data.img_db_range);
-            assertTrue(~obj.fh_range_check(img_db_range_diff,0.001));
+            assertTrue(obj.fh_range_check(img_db_range_diff,1.e-7));
```
Comment (oerc0122) [inline _test/test_sqw/test_mask.m:262]: Tightening these tolerances?

Comment (oerc0122) [reply _test/test_sqw/test_mask.m:262]: Might want to switch these to the "official" [1.e-7, 0]

File: _test/test_sqw/test_mask.m [hunk 2]
```diff
@@ -293,7 +293,9 @@ function test_mask_pixels_removes_pixels_given_in_mask_array(obj)
             
             assertEqual(new_sqw.data.pix.num_pixels, sum(mask_array));
             assertFalse(equal_to_tol(new_sqw.data.s, sqw_obj.data.s, -1e-4));
-            assertFalse(equal_to_tol(new_sqw.data.img_db_range, sqw_obj.data.img_db_range, -1e-4));
+            % masking have not changed binning, so img_db_range remains the
+            % same
+            assertTrue(equal_to_tol(new_sqw.data.img_db_range, sqw_obj.data.img_db_range, -1e-4));
```
Comment (oerc0122) [inline _test/test_sqw/test_mask.m:298]: Likewise [0, 1.0e-4]

**Review by oerc0122 — APPROVED** [review]

File: _test/test_sqw/test_mask.m (hunk 1 above)
Comment (abuts) [reply _test/test_sqw/test_mask.m:262]: The behavior have changed. Masking does not change img_db_ranges as this is the range where pixels were rebinned into, so masking would not change it unless you rebin. 
 `fh_range_check` does not recognizes [1.e-7,0].

Comment (oerc0122) [reply _test/test_sqw/test_mask.m:262]: Sorry, I thought it would use `equal_to_tol` underneath.

**Pull request merged**